snakemake/script/__init__.py

```
@@ -42,6 +44,56 @@
+        if self._url_args:
+
+            def fmt_arg(key, value):
+                return f"{key}={urllib.parse.quote(value)}"
+
+            args = f"?{'&'.join(fmt_arg(key, value) for key, value in self._url_args.items())}"
+        else:
+            args = ""
+        if self._anchor:
+            anchor = f"#{urllib.parse.quote(self._anchor)}"
+        else:
+            anchor = ""
```

Ruff (0.8.0)

Use ternary operator `anchor = f"#{urllib.parse.quote(self._anchor)}" if self._anchor else ""` instead of `if`-`else`-block

Replace `if`-`else`-block with `anchor = f"#{urllib.parse.quote(self._anchor)}" if self._anchor else ""`

(SIM108)